core/app/models/spree/wished_product.rb

```diff
@@ -0,0 +1,16 @@
+module Spree
+  class WishedProduct < Spree::Base
```

Comment:
@MatthewKennedy I think next step is to rename this to `WishedVariant` as the name is misleading ATM